notebooks/VAD

```diff
@@ -0,0 +1,37 @@
+from scipy import signal
+# input : y - the signal, sr - sample rate
+# output : start_times - vector of start times of the sound events
+#          end_time - vector of end times of the sound events            
+def detect_activity(y, sr,
```

Comment:
It would be good to add a docstring describing the role of each parameter.
For `y`, `sr`, etc. up to `eps`, you may copy-paste the librosa docstring of the `pcen` function

For `medfilt_time_constant` up to `activity_threshold`, please write your own description in the same style